supertokens_python/recipe/emailpassword/syncio/__init__.py

```diff
@@ -31,7 +31,7 @@ def verify_email_using_token(token: str, user_context: Union[None, Dict[str, Any
     return sync(verify_email_using_token(token, user_context))
 
 
-async def unverify_email(user_id: str, user_context: Union[None, Dict[str, Any]] = None):
```

Comment:
This fix should go in another PR to the main x.y branch as it is unrelated to this PR.

Reply:
Done.

Reply:
You need to make another PR fixing this (to the latest X.Y branch).

Reply:
Done.